Project.toml

```diff
@@ -6,23 +6,16 @@ version = "0.2.15"
 [deps]
 Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
 HypertextLiteral = "ac1192a8-f4b3-4bfe-ba22-af5b92cd3ab2"
-LaTeXStrings = "b964fa9f-0449-5b57-a5c2-d3ea65f4040f"
 Latexify = "23fbe1c1-3f47-55db-b15f-69d7ec21a316"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 PlutoLinks = "0ff47ea0-7a50-410d-8455-4348d5de0420"
 PlutoUI = "7f904dfe-b85e-4ff6-b463-dae2292396a8"
-Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 
 [compat]
+Downloads = "<0.0.1, 1"
```

Comment:
What does v <0.0.1 mean?  Why not just set this to "1"?  Same for Markdown

Reply:
Markdown.jl and Downloads.jl are standard libraries and used to be coupled to their Julia releases. Now they are stand-alone packages and have version numbers. For backward compatibility, these new compat entries have to include `<0.0.1`. The `1` is the latest stable major release of Downloads.jl (currently `1.6.0`).

You can find out more here: https://discourse.julialang.org/t/psa-compat-requirements-in-the-general-registry-are-changing/104958